manifests/params.pp

```diff
@@ -114,12 +114,14 @@
   case $::operatingsystem {
     'RedHat', 'CentOS', 'Fedora', 'Scientific', 'Amazon', 'OracleLinux', 'SLES', 'OpenSuSE': {
       # main application
-      $package = [ 'logstash' ]
+      $package = [ 'logstash' ],
```

Comment:
Looks like a typo here.

Please have a look at the output from the acceptance tests:

https://travis-ci.org/elastic/puppet-logstash/jobs/169621652

Thanks.